### ISSUE TITLE
Adding the ability to select the face to be swapped with in the video that contains more than one face

### ISSUE THREAD

**Issue by dodo295** [neuralchen/SimSwap#18]

**Comment by NNNNAI** [comment]
We will consider adding this ability to the current project. Many thanks.

**Comment by AlonDan** [comment]
**Question**:
With the current version, how it's choosing automatically which face to swap? (based on 1 face video)
Is it the biggest / brightest one? is it most Left to Right or something combine?

Just wondering until we'll be able to choose faces, respectfully to your hard work if it will be a thing of course.

**Comment by NNNNAI** [comment]
Technically speaking, the subject of each face change is the person with the highest confidence in the face detector. In layman's terms, it is the most prominent face.

**Comment by NNNNAI** [comment]
Thanks for your comments. The function of appointing a specific person to change face has been added, please go to [Inference for image or video face swapping](https://github.com/neuralchen/SimSwap/blob/main/docs/guidance/usage.md) for details. If this rep helps you a lot, please star it! Many thanks~.

**Comment by robbsaber** [comment]
> Inference for image or video face swapping

Will this be added to the colab notebook?

**Comment by AlonDan** [comment]
Thanks for adding this feature,, I just quick tested it and it's GREAT! you rocks❤

I just added an issue which is a suggestion to help in the project.
consider I'm not a programmer but I fall in love with this project already so I want to help using my skills if possible.

You can read about it here: #32

**Comment by NNNNAI** [comment]
> > Inference for image or video face swapping
> 
> Will this be added to the colab notebook?

We will make colab demos of all the new features recently. Many thanks~.

**Comment by NNNNAI** [comment]
@robbsaber the colab of "multi specific " has been released, please go to [colab](https://colab.research.google.com/github/neuralchen/SimSwap/blob/main/MultiSpecific.ipynb) for details.
